### PR TITLE
PRO-3623 - Add purchase_order_number to projects

### PR DESCRIPTION
**Pull request by Pegasuz**
Do not merge before implementation is merged!

This PR adds the "purchase_order_number" to the projects.info, projects.create and projects.update endpoints.

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -3201,6 +3205,7 @@ Get details for a single project.
                         + Members
                             + decision_maker
                             + member
+            + purchase_order_number: `000023` (string)
```
Comment (joaoinacio) [inline apiary.apib]: It should be `nullable`, IIRC?

**Review by adognameddog — CHANGES_REQUESTED** [review]
Purchase order number should be nullable, like @joaoinacio mentioned. Otherwise looking good!

**Review by ketlane — DISMISSED** [review]

File: apiary.apib [hunk 2]
```diff
@@ -374,6 +374,10 @@ We list all backwards-compatible additions here. These are currently available i
 
 #### Latest
 
+- We added the `purchase_order_number` field to `projects.info`, `projects.create`, and `projects.update`
+
+#### April 2020
```
Comment (ketlane) [inline apiary.apib:379]: Not sure we should add the month grouping in this PR, but I'm not sure we shouldn't either. Maybe there are other opinions about it?

**Review by ArnaudWeyts — DISMISSED** [review]

File: apiary.apib (hunk 2 above)
Comment (adognameddog) [reply apiary.apib:379]: If someone adds new documentation in June, then that person should add a new header and place everything from latest in there.

**Review by adognameddog — DISMISSED** [review]

**Review by mikeverf — DISMISSED** [review]

**Review by joaoinacio — DISMISSED** [review]

**Review by adognameddog — APPROVED** [review]

**Review by mikeverf — APPROVED** [review]

**Review by ArnaudWeyts — APPROVED** [review]

**Pull request merged**